scripts/attempt-backport.js

```diff
@@ -27,9 +32,9 @@ module.exports = function (app) {
     const options = { owner, repo, prId, logger: event.logger }
 
     debug(`/${owner}/${repo}/pull/${prId} sync`)
-    queueAttemptBackport(options, 7, false)
-    queueAttemptBackport(options, 6, true)
-    queueAttemptBackport(options, 4, true)
+    nodeVersions.forEach(function(node) {
+      queueAttemptBackport(options, node.version, !!node.lts)
```

Comment:
Minor nitpick, this line is identical to [line #51](https://github.com/nodejs/github-bot/pull/93/files#diff-eea09a4f0f8288ff9bd49ef44625f0c1R51) -- maybe even the body of this `forEach()` could be extracted for further DRY-iness.

Otherwise this LGTM.